packages/medusa/src/strategies/batch-jobs/product/import.ts

```diff
@@ -702,22 +704,29 @@ class ProductImportStrategy extends AbstractBatchJobStrategy {
 
         uploadPromises.push(promise)
 
+        files[op] = fileKey
         writeStream.write(JSON.stringify(results[op]))
         writeStream.end()
       }
     }
 
+    await this.batchJobService_
+      .withTransaction(transactionManager)
+      .update(batchJobId, {
+        result: { files },
+      })
+
     await Promise.all(uploadPromises)
   }
 
   /**
-   * Remove parsed ops JSON file.
+   * Download parsed ops JSON file.
    *
-   * @param batchJobId - An id of the current batch job being processed.
+   * @param batchJob - the current batch job being processed
    * @param op - Type of import operation.
    */
   protected async downloadImportOpsFile(
-    batchJobId: string,
+    batchJob: BatchJob,
```

Comment:
comment: Breaking change. Do we want to add backward compatibility or push a minor? Wdyt?

Reply:
I'm for a minor change since the new way of handling filekeys for ops files is more future proof

Reply:
Let's do it 👍